automation/taskcluster/lib/tasks.py

```diff
@@ -160,6 +149,16 @@ def _craft_build_ish_task(
         routes = [] if routes is None else routes
         features = {} if features is None else features
 
+        checkout_command = (
+            "export TERM=dumb && git fetch {} {} --tags && "
```

Comment:
FYI: "export TERM=dumb" has been set in our Docker images for a while already :)

Reply:
Oh right! Fixed